README.md

```diff
@@ -1,11 +1,54 @@
-## My Project
+## ACK Multi-Controller Helm Chart
 
-TODO: Fill this README out!
+The ACK Multi-Controller Helm chart, shortened to `ack-chart`, acts as a wrapper
+for all ACK controllers released under General Availability (GA). 
 
-Be sure to:
+`ack-chart` lists each ACK controller as a subchart dependency.
 
-* Change the title in this README
-* Edit your repository description on GitHub
+`ack-chart` is currently in **developer preview** and is subject to breaking
+changes.
+
+## Usage
+
+Helm 3.8+ is required to install `ack-chart`, since it is hosted in an
+[OCI-based image registry][oci-registry].
+
+To install the latest version of `ack-chart`, use the following command:
+```bash
+export LATEST_VERSION=`curl -sL https://api.github.com/repos/aws-controllers-k8s/ack-chart/releases/latest | grep '"tag_name":' | cut -d'"' -f4`
+helm install --create-namespace -n ack-system ack-chart \
+  oci://public.ecr.aws/aws-controllers-k8s/ack-chart --version=$LATEST_VERSION
+```
+
+[oci-registry]: https://helm.sh/docs/topics/registries/
+
+### Enabling Subcharts
+
+By default, when installing the `ack-chart`, **none of the subcharts are
+installed**. To enable a subchart, set the value associated with the ACK
```

Comment:
Be pedantic here:  "...set the value (in `values.yaml`) associated with...."

Reply:
Well to be really pedantic, there are multiple ways to set the value. You can also do it on the command line using `--set <>=<>` as well, which is why I don't prescribe a method for doing it.

Reply:
Great, then mention both options!